src/server/services/secrets/views/environment.njk

```diff
@@ -46,9 +46,9 @@
   {% endset %}
 
   {{ serviceSecretsTableRows.push([
-    { html: appTag({ text: secret.key, classes: "govuk-tag--green" }) },
-    { html: iconHtml, classes: "app-!-width-one-tenth" },
-    { html: actionHtml, classes: "app-!-width-one-tenth app-!-align-vertical-middle" }
+    { html: appTag({ text: secret.key, classes: "govuk-tag--green", attributes: {"data-testid": "app-secret-cell-"+secret.key.toLowerCase()} }) },
+    { html: iconHtml, classes: "app-!-width-one-tenth", attributes: {"data-testid": "app-secret-status-"+secret.key.toLowerCase()}  },
+    { html: actionHtml, classes: "app-!-width-one-tenth app-!-align-vertical-middle", attributes: {"data-testid": "app-secret-update-"+secret.key.toLowerCase()} }
```

Comment:
Worth noting there is a `lower` in nunjucks - https://mozilla.github.io/nunjucks/templating.html#lower

So this could become:
```
"data-testid": "app-secret-update-{{ secret.key | lower }}"}
```